src/emscripten-optimizer/parser.cpp

```diff
@@ -41,6 +41,7 @@ IString TOPLEVEL("toplevel"),
         CONTINUE("continue"),
         SWITCH("switch"),
         STRING("string"),
+        TRY("try"),
```

Comment:
I see, thanks. I think we should add tests for this new functionality, see for example #1126 which adds `--emit-js-wrapper` for the new fuzzer and adds reference tests for that output.